packages/spanner/src/transaction-request.js

```diff
@@ -201,6 +202,12 @@ TransactionRequest.prototype.createReadStream = function(table, query) {
     table: table
   }, query);
 
+  if (this.transaction && this.id) {
```

Comment:
Is this the entire bugfix? :open_mouth: 
As someone unfamiliar with the code, can you explain why this was the solution? It is not really clear from reading the changelog.